contents/staticAnalysis/intro.md

```diff
@@ -0,0 +1,92 @@
+## Static Analysis
+Static analysis is the process of analysing computer programme **without** executing the problem. This practice is often used to ensure the codes follow certain structures or standards (e.g coding standards).
+
+It is possible to do static analysis manually, but there are automated tools(static analysers) that can assist developers in this process. 
+
+### Why Static Analysis?
+
+#### Analyse Thoroughly
+In some situations, it is impossible to write all test cases to achieve 100% test coverage. There will be some places in the code that are not covered by test cases, which may result in bugs. In static analysis, all the related files/codes will be analysed.
+
+#### Enforce a Standard in Project
+Many projects enforce certain coding standards. For example, some project require the following format for if statement.
+
+``` java
+if (condition) {
+	// true
+} else {
+	// false
+}
+``` 
+
+While others enforce the following standard
+
+``` java
+if (condition) {
+	// true
+}
+else {
+	// false
+}
+```
+Such standards can be configured and forced in static analysis.
+
+#### Find Potential Bugs Early
```

Comment:
maybe you can shift this above "Enforce a Standard in Project" to show how static analysis is an alternative to test cases